webapp/templates/base.html.twig

```diff
@@ -16,14 +16,13 @@
     {% for file in customAssetFiles('js') %}
         <script src="{{ asset('js/custom/' ~ file) }}"></script>
     {% endfor %}
+    {% for file in customAssetFiles('css') %}
+        <link rel="stylesheet" href="{{ asset('css/custom/' ~ file) }}">
+    {% endfor %}
```

Comment:
Should we move this to BELOW the extrahead block? Now things in style_domjudge.css will overwrite the custom CSS instead of the other way around

Reply:
That does make more sense, also because that's what is mentioned in the documentation.